src/sonic-config-engine/portconfig.py

```diff
@@ -106,10 +106,15 @@ def get_port_config(hwsku=None, platform=None, port_config_file=None, hwsku_conf
             ports = ast.literal_eval(json.dumps(port_data))
             port_alias_map = {}
             port_alias_asic_map = {}
+            port_name_to_index_map = {}
             for intf_name in ports.keys():
                 if "alias" in ports[intf_name]:
                     port_alias_map[ports[intf_name]["alias"]] = intf_name
-            return (ports, port_alias_map, port_alias_asic_map)
+                if "alias" in ports[intf_name] and "asic_port_name" in ports[intf_name]:
```

Comment:
@SuvarnaMeenakshi why in same if confition ? if asic_port name is not there even alias will not get populated